### ISSUE TITLE
[Help/Suggestion] Claimed Chunk entering/leaving

### ISSUE THREAD

**Issue by Ashendale** [thexaero/open-parties-and-claims#162]
I'm very sorry if this is the wrong place to ask but, if it is and/or if I'm being dumb, then this would be a suggestion.

When entering or leaving a claimed you get a notification at the bottom of the screen saying "Person's Claim" or "Wilderness" (as you enter or leave that area, respectively). I wish to turn it off, but for the life of me, I cannot find the option. If it exists.

If it doesn't exit, I'd like to suggest for it to be there.

The map/minimap is configured to show me the state of the chunks, and I'd prefer to consult them than to have a distracting notification.

Thank you for your time and efforts.

**Comment by thexaero** [comment]
I'll consider adding a toggle.

**Comment by abrossimow** [comment]
Jumping on this necro ship: have you considered? :)

**Comment by DoktorBarron** [comment]
Hoping there's any new news on this, because constantly going between wilderness, a normal claim, and a force loaded claim, that popup is driving me nutty. 

- edit
I genuinely for the love of me cannot find where the option is.

**Comment by thexaero** [comment]
Yes, this has been added a long time ago. There was a duplicate issue for the same thing.

**Comment by DoktorBarron** [comment]
> Yes, this has been added a long time ago. There was a duplicate issue for the same thing.

May I please ask where such an option is in the menu or called?, I've been unable to find what the option is.
This is where the search engines are pushing me (and most certainly others) when I (or they) try an investigate having the same predicament. 

it'd be a massive help if this information was shared. Thank you.
